py4DSTEM/io/nonnative/read_dm.py

```diff
@@ -4,16 +4,15 @@
 from pathlib import Path
 from ncempy.io import dm
 from ..datastructure import DataCube, Array, Metadata
-#from ...preprocess.utils import bin2D
+from ...process.utils import bin2D
 
 
```

Comment:
This looks correct and doesn't raise any import errors but I haven't checked the against un/accepted file types